SPECS/kernel/config

```diff
@@ -1,19 +1,19 @@
 #
```

Comment:
Note/reminder: Can you please disable CONFIG_GCC_PLUGIN_RANDSTRUCT to resolve the issue with kernel dumps